src/helpers/useErrorHandler.js

```
@@ -0,0 +1,16 @@
+import { useState, useMemo } from 'react'
+
+export const useErrorHandler = () => {
+    const [error, setError] = useState('')
+    const errorHandler = useMemo(
+        () => event => {
+            console.log('event', event)
+            try {
+                setError(event.target.response.message)
+            } catch (e) {}
```

A comment here explaining why it's ok to swallow this error would be nice.